Projects/App/Sources/Common/SplashViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SplashViewModel.swift
+//  GGiriGGiri
+//
+//  Created by 카카오vx on 2022/08/15.
+//  Copyright © 2022 dvHuni. All rights reserved.
+//
+
+import Foundation
+
+import RxRelay
+import RxSwift
+import UIKit
+
+protocol SplashViewModelProtocol {
+    var deadlineDataUpdated: PublishRelay<Int> { get }
+}
+
+final class SplashViewModel: SplashViewModelProtocol {
+    
+    private let disposeBag = DisposeBag()
+    private let gifticonService: GifticonService
+    
+    init(network: Networking) {
+        self.gifticonService = GifticonService(network: network)
+        
+        deadlineInfo()
+    }
+    
+    var deadlineDataUpdated = PublishRelay<Int>()
+    
+    private func deadlineInfo() {
+        gifticonService.deadline(.init(orderBy: .deadLine, category: .all))
+            .subscribe { [weak self] responseModel in
+                guard let responseModel = responseModel.data else {
+                    return
+                }
+                let entity = GifticonEntity.init(responseModel)
+                self?.deadlineDataUpdated.accept(entity.gifticonList.count)
+                
+                self?.changeRootViewController(deadlineData: entity.gifticonList.count)
+            } onFailure: { error in
+                print(error.localizedDescription)
+            }.disposed(by: disposeBag)
+    }
+    
+    private func changeRootViewController(deadlineData: Int) {
+        
+        var deadlineDataExist = true
```

Comment:
```suggestion
        var isDeadlineDataExist = true
```

Bool타입같아서 요건 어떠신지요..?!